cmake/ElektraCache.cmake

```diff
@@ -185,6 +185,42 @@ set(TOOLS ${TOOLS} CACHE STRING ${TOOLS_DOC} FORCE)
 
 
 #
+# List of IO-Wrappers to build
+#
+
+remember_for_removal(IOWRAPPERS TO_REMOVE_IOWRAPPERS)
```

Comment:
This macro is a crude cmake hack and was (for good reasons) already replaced for plugins (bindings and tools unfortunately still use it). Would be better if we do not rely new stuff on it. Maybe you can simply have the iowrappers as plugins or bindings?

Alternatively, we should introduce a `LIBS` variable.

Reply:
Good point! I will use BINDINGS like the Swig bindings do (e.g. io_uv, io_glib, ALL and IO shortcut)